### PR TITLE
Move checkout

### PR DESCRIPTION
**Pull request by jakedurell**
1.     VLA headline in extension popup – keep users up to date with VLA news

2.     New settings menu with settings in one place
- New VLA logo/footer checkbox – ensure generic logo and custom footer is overridden when preparing for VLA
- Attorney/Preparer name, address, and phone
- Preparer role (see role text in cover letter below - #5)
- Custom Footer (applied when VLA logo option is not checked)

3.     All settings from #2 moved to local storage (will not be erased when browser is closed or when computer is restarted)

4.     Removed “Ineligible” option. Ensured that “No Filing” appears on checkout sheet.

5.     Add cover letter

- “Scroll to” Link on left Nav
- Logo based on VLA selection in settings
- Address block
- List of all filings
- Role text (based on whether preparer is attorney, entering appearance, admin or state)
- County contact info (See #6)

6.     Remotely hosted adminConfig file
(allows updating content instantly for all ExpungeVT users)
- Expunge Headline (See #1)
- RoleText - update default letter text as requested
- County contact – put current county contact info on cover letter

7.     Move checkout sheet to top and redesign
- Stipulation legend
- Remove pro se text (added to letter)
- Add arrest and offense dates
- All counts appear on checkout even if no action taken (unless deleted)

8.     Remove extra checkout sheet

9.     Modifications for longer names
- Adjusted caption (lines instead of parenthesis)
- Top padding and lower z-index on left nav to allow for header room

10.  Remove remotely hosted code (CDN links)
- A security measure which may allow for faster publishing times and faster load times.

### CODE REVIEW

**Comment by jakedurell** [comment]
@jeremyvermont -This is still a work in progress. Please look at branch if you have a sec. Something funny is happening on the print view where the Noa starts in the middle. Let me know if you have a chance for a quick call in the next week or so.

image

**Comment by jakedurell** [comment]
page break is resolved with last commit.

**Comment by jeremyvermont** [comment]
Things are working good overall! There is a UI regression on the start screen. The buttons are now taller and the icon is wrapping to a new line.

Before:
Screen Shot 2020-04-28 at 8 01 01 PM

After:
Screen Shot 2020-04-28 at 8 01 52 PM

**Comment by jakedurell** [comment]
@jeremyvermont - Thanks for your review. I fixed that issue which was just a problem with the newer version of bootstrap. I set the npm install to the CDN link we had before. 

I have a few more things I want to do on this branch. I am going to delete the node modules now that I have package.json. It will make it easier for you to review the PR. 

I am also going to add some things to the JSON, such as cover letter content, so that VLA can adjust content without having to republish the extension.

**Comment by jeremyvermont** [comment]
Sounds good, let me know when you want me to take another look.

File: extensionDirectory/filings.css [hunk 1]
```diff
@@ -93,6 +93,9 @@ a.btn {
     margin-left:0.5rem;
 }
 
+.card-header__select .form-control{
```
Comment (jeremyvermont) [inline extensionDirectory/filings.css:96]: @jakedurell Can you remove this empty css rule

File: README.md [hunk 2]
```diff
@@ -16,13 +16,15 @@ This project works through a chrome extension that reads an HTML docket sheet fr
...
-- Users must have an active subscription with [VT Courts Online](https://secure.vermont.gov/vtcdas/user)
-- This tool does not save or expand access to any data
+- Users must have an active subscription with [VT Courts Online](https://secure.vermont.gov/vtcdas/user).
+- This tool does not save or expand access to any data captured from court records.
 
 ## Setup for local development
 
 1. Clone the github repository to your machine
 
+1. In your terminal, navigate to the `extensionDirectory` folder and run NPM install.
```
Comment (jeremyvermont) [inline README.md:26]: can you change NPM install to `npm install`. Also, it appears that the all of these steps are step 1.

Comment (jakedurell) [reply README.md:26]: @jeremyvermont the "1s" are just markdown language. If you right-click the file and click preview, you will see. You can see on github on master as well. I changed the NPM.

File: extensionDirectory/filings.css (hunk 1 above)
Comment (jakedurell) [reply extensionDirectory/filings.css:96]: @jeremyvermont I removed the empty CSS and the padding on body so the double border went away. I made sure the box and empty object did not appear when offline or when the headline is empty.

**Comment by jeremyvermont** [comment]
Re-tested, looks good!

**Pull request merged**